compiler/src/dotty/tools/repl/Rendering.scala

```diff
@@ -70,10 +72,26 @@ private[repl] class Rendering(compiler: ReplCompiler,
   /** Render value definition result */
   def renderVal(d: Denotation)(implicit ctx: Context): Option[String] = {
     val dcl = d.symbol.showUser
-    val resultValue =
-      if (d.symbol.is(Flags.Lazy)) Some("<lazy>")
-      else valueOf(d.symbol)
 
-    resultValue.map(value => s"$dcl = $value")
+    try {
+      val resultValue =
+        if (d.symbol.is(Flags.Lazy)) Some("<lazy>")
+        else valueOf(d.symbol)
+
+      resultValue.map(value => s"$dcl = $value")
+    }
+    catch { case ex: InvocationTargetException => Some(renderError(ex)) }
+  }
+
+  /** Render the stack trace of the underlying exception */
+  private def renderError(ex: InvocationTargetException): String = {
+    val cause = ex.getCause match {
+      case ex: ExceptionInInitializerError => ex.getCause
```

Comment:
I am trying to isolate the underlying exception. This is based on nothing but experiment:
```scala
> null.toString
Exception in thread "main" java.lang.reflect.InvocationTargetException
	...
Caused by: java.lang.ExceptionInInitializerError
	...
Caused by: java.lang.NullPointerException
	...
```
```scala
scala> def foo: Int = foo + 1 
def foo: Int
scala> foo 
Exception in thread "main" java.lang.reflect.InvocationTargetException
	...
Caused by: java.lang.StackOverflowError
	...
```
I am sure there is a better a way to do it.

Reply:
We really need tests for all of those :)

Reply:
Working on it